itk-module.cmake

```diff
@@ -3,15 +3,15 @@
 get_filename_component(MY_CURRENT_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 file(READ "${MY_CURRENT_DIR}/README.rst" DOCUMENTATION)
 
-# itk_module() defines the module dependencies in Mesh3DProcrustesAlignFilter
-# Mesh3DProcrustesAlignFilter depends on ITKCommon
-# The testing module in Mesh3DProcrustesAlignFilter depends on ITKTestKernel
-# and ITKMetaIO(besides Mesh3DProcrustesAlignFilter and ITKCore)
+# itk_module() defines the module dependencies in ITKShape
+# ITKShape depends on ITKCommon
+# The testing module in ITKShape depends on ITKTestKernel
+# and ITKMetaIO(besides ITKShape and ITKCore)
 # By convention those modules outside of ITK are not prefixed with
 # ITK.
 
 # define the dependencies of the include module and the tests
-itk_module(Mesh3DProcrustesAlignFilter
+itk_module(ITKShape
```

Comment:
Encountering an error when changing this line from `ITKShape` to `Shape`:
> CMake Error at C:/repos/ITK/CMake/ITKModuleKWStyleTest.cmake:52 (file):
  file failed to open for writing (Permission denied):

    /ITKKWStyleFiles.txt
Call Stack (most recent call first):
  C:/repos/ITK/CMake/ITKModuleMacros.cmake:196 (itk_module_kwstyle_test)
  C:/repos/ITK/CMake/ITKModuleExternal.cmake:162 (itk_module_impl)
  CMakeLists.txt:7 (include)

Tracing to [line 52 in `ITKModuleKWStyleTest.cmake`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/CMake/ITKModuleKWStyleTest.cmake#L52) it looks like `${Shape_BINARY_DIR}` is not being set, which I've confirmed with CMake debugging statements. Unclear on why this is not being generated appropriately. Related lines:
> set(_kwstyle_itk_module_files_list_file  "${${itk-module}_BINARY_DIR}/ITKKWStyleFiles.txt")
    set(_kwstyle_file_list "${${itk-module}_SOURCE_DIR}/include/itk*.h"
                            "${${itk-module}_SOURCE_DIR}/include/itk*.hxx"
                            "${${itk-module}_SOURCE_DIR}/src/*.cxx"
                            "${${itk-module}_SOURCE_DIR}/src/*.h"
                            "${${itk-module}_SOURCE_DIR}/src/*.hxx"
                            "${${itk-module}_SOURCE_DIR}/test/*.cxx")
    file(WRITE ${_kwstyle_itk_module_files_list_file} "")

Reply:
@thewtex if you have a chance could you please weigh in on this issue? This is blocking ITKShape from being added as an ITK remote module.

Reply:
Resolved. As expected I needed to change `ITKShape` to `Shape` here, the error I received after the change was related to Python wrappings and not directly to the module name change.